app/scripts/controllers/transactions/pending-tx-tracker.js

```diff
@@ -209,9 +209,9 @@ export default class PendingTransactionTracker extends EventEmitter {
    */
   async _checkIfTxWasDropped (txMeta) {
     const { hash: txHash, txParams: { nonce, from } } = txMeta
-    const networkNonce = await this.query.getTransactionCount(from)
+    const networkNextNonce = await this.query.getTransactionCount(from)
 
-    if (parseInt(nonce) > parseInt(networkNonce)) {
+    if (parseInt(nonce) >= parseInt(networkNextNonce)) {
```

Comment:
Unrelated but can we add radix values to these calls:

```suggestion
    if (parseInt(nonce, 16) >= parseInt(networkNextNonce, 16)) {
```

(I think these are hex?)

Reply:
`networkNextNonce` definitely is. I'll verify the other one is as well.

Reply:
This has been done. I've added this comment to the PR description and commit as explanation:

>`parseInt` is now called with an explicit radix of `16` as well, to ensure both nonce strings are always parsed as hex. In all cases I am aware of, these nonce strings were prefixed by `0x`, meaning that `parseInt` would default to a radix of `16`, so this likely doesn't constitute a functional change.